OWNERS

```diff
@@ -1,30 +1,35 @@
 approvers:
+  - Luap99
+  - TomSweeneyRedHat
+  - ashley-cui
   - baude
   - edsantiago
+  - flouthoc
   - giuseppe
   - jwhonce
-  - Luap99
+  - lsm5
   - mheon
   - mtrmac
   - rhatdan
   - saschagrunert
-  - TomSweeneyRedHat
   - umohnani8
   - vrothberg
   - zhangguanzhang
 reviewers:
-  - ashley-cui
```

Comment:
Do approvers still have review ability?

Reply:
I think so. I am not really sure that approvers need to be in both locations.